certified-connectors/tegolySIGN/apiProperties.json

```diff
@@ -0,0 +1,52 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "clientId": "a2b9e741-b3c9-49ab-b278-1f2c21eacf5c",
+          "scopes": [
+            "api://a2b9e741-b3c9-49ab-b278-1f2c21eacf5c/PATM"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "AzureActiveDirectoryResourceId": "https://management.core.windows.net/",
+            "IsOnbehalfofLoginSupported": true
+          },
+          "customParameters": {
+            "loginUri": {
+              "value": "https://login.windows.net"
+            },
+            "tenantId": {
+              "value": "common"
+            },
+            "resourceUri": {
+              "value": "https://management.core.windows.net/"
+            },
+            "enableOnbehalfOfLogin": {
+              "value": "false"
+            }
+          }
+        }
+      },
+      "token:TenantId": {
+        "type": "string",
+        "metadata": {
+          "sourceType": "AzureActiveDirectoryTenant"
+        },
+        "uiDefinition": {
+          "constraints": {
+            "required": "false",
+            "hidden": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#316bac",
+    "capabilities": [],
+    "publisher": "tegoly GmbH"
```

Comment:
@JanGosda please add the stackOwner property. 
"stackOwner": "Name of the owner of the service or product." should be the same of Publisher in your case.